test/vm.install

```diff
@@ -51,6 +52,11 @@ if [ "$PLATFORM_ID" = "platform:el10" ]; then
     printf  '[network]\nfirewall_driver = "nftables"\n' > /etc/containers/containers.conf
 fi
 
+if [ "$ID" = "arch" ]; then
+    # Disable PerSourcePenalties, our CI is too fast
+    echo "PerSourcePenalties no" >>/etc/ssh/sshd_config
+fi
```

Comment:
Shouldn't this happen in bots?

Reply:
I'm torn. In general I try to keep our VM images as close to reality as possible, but: this particular feature is very specifically messing with CI, in a non-obvious way, and most people are not aware of it. Our other projects don't currently seem to be affected by it? (Perhaps c-machines migration, but I suppose we don't cover a lot of failed SSH attempts there?)

On the other hand, this is most likely *not* arch specific, but a feature of OpenSSH in general, and this will trickle into  So my recommendation is to drop the ID == arch check, write into a separate conf file to reduce the chance of messing up the default file, and land it in cockpit. We can change it in bots if/when it starts to affect other projects. 

WDYT?

Reply:
> So my recommendation is to drop the ID == arch check,

As Marius noticed, that doesn't work with "older" versions.